Formula/c-kermit.rb

```diff
@@ -61,3 +68,1264 @@ def install
      x = (int) ((stdin->_IO_read_end) - (stdin->_IO_read_ptr));
      debug(F101,"cmdconchk _IO_file_flags","",x);
  #else  /* _IO_file_flags */
+
+diff --git a/ck_ssl.c b/ck_ssl.c
```

Comment:
Where is this patch from?

Reply:
Based on https://github.com/BAN-AI-Communications/c-kermit/commit/fd0d853d76227d7929cabb63cae6ba57b295f633.diff which overall is similar to upstreams changes.  (It's not trivially possible to directly diff against the upstream simply there have been extensive changes over the last 12 years. The upstream changelog is at https://kermitproject.org/ckupdates.html where the first 50% describes the differences from this release to the current state.)

Reply:
I think we should just wait for an upstream release instead of cherry picking commits for extra features.

Reply:
See https://github.com/Homebrew/homebrew-core/pull/130987#issuecomment-1547405456 for what I think is a better patch to append.